fastlane/Fastfile

```diff
@@ -191,11 +191,28 @@ end
   # bundle exec fastlane new_hotfix_release version:10.6.1
   # bundle exec fastlane new_hotfix_release skip_confirm:true version:10.6.1
   #####################################################################################
-  desc "Creates a new hotfix branch from the given tag"
+  desc 'Creates a new hotfix branch for the given version:x.y.z. The branch will be cut from the tag x.y of the previous release'
   lane :new_hotfix_release do | options |
     prev_ver = ios_hotfix_prechecks(options)
     ios_bump_version_hotfix(previous_version: prev_ver, version: options[:version])
-    ios_tag_build()
+  end
+
+  #####################################################################################
+  # finalize_hotfix_release
+  # -----------------------------------------------------------------------------------
+  # This lane finalizes the hotfix branch.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane finalize_hotfix_release [skip_confirm:<skip confirm>]
+  #
+  # Example:
+  # bundle exec fastlane finalize_hotfix_release skip_confirm:true
+  #####################################################################################
+  desc 'Performs the final checks and triggers a release build for the hotfix in the current branch'
+  lane :finalize_hotfix_release do |options|
+    ios_finalize_prechecks(options)
+    version = ios_get_app_version
+    trigger_release_build(branch_to_build: "release/#{version}")
```

Comment:
Also happy to merge with the "fixed" name, and to take ownership of following up with a PR that aligns the names, since I was the one that made them diverge in the first place.

Reply:
I agree on aligning the lane names.
I concur with your name being clearer, but I think at that stage given we already have precedent in all other repos, it would be easier to migrate that repo to `trigger_release_build` rather than the other way around.

Reply:
+1 

How do you feel about merging this PR as it is (with the fix of course) and for me to followup with the naming alignment?

Reply:
Works for me 👍

Reply:
Fixed in 905a50c

Reply:
![](https://media.giphy.com/media/10LNj580n9OmiI/source.gif)